src/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
-# (C) British Crown Copyright 2024 Met Office
+# (C) British Crown Copyright 2024-2025 Met Office
```

Comment:
We aren't updating these ad-hoc I think it was centrally decided there is no need. We can update them in bulk if you would really like to, but my understanding is that legally they don't mean much as long as we have the license in place.